libs/CCAppCommon/src/ccApplicationBase.cpp

```diff
@@ -48,8 +48,8 @@
 
 void ccApplicationBase::initOpenGL()
 {
-	//See http://doc.qt.io/qt-5/qopenglwidget.html#opengl-function-calls-headers-and-qopenglfunctions
-	/** Calling QSurfaceFormat::setDefaultFormat() before constructing the QApplication instance is mandatory
+	//! See http://doc.qt.io/qt-5/qopenglwidget.html#opengl-function-calls-headers-and-qopenglfunctions
```

Comment:
You shouldn't add //! in a cpp file (that's only for short descriptions of methods or members)

Here the intent was to add a comment block (`/* ... */ `would have been sufficient though)